packages/trip-details/src/index.tsx

```diff
@@ -218,12 +201,17 @@ export function TripDetails({
 
   const departureDate = new Date(itinerary.startTime);
 
-  // Compute calories burned.
-  const {
-    bikeDuration,
-    caloriesBurned,
-    walkDuration
-  } = coreUtils.itinerary.calculatePhysicalActivity(itinerary);
+  // Compute total time spent active.
+
+  let walkDurationSeconds = 0;
+  let bikeDurationSeconds = 0;
+  itinerary.legs.forEach(leg => {
+    if (leg.mode.startsWith("WALK")) walkDurationSeconds += leg.duration;
+    if (leg.mode.startsWith("BICYCLE")) bikeDurationSeconds += leg.duration;
+  });
```

Comment:
This can probably be cleaned up into 2 reducers (maybe even with a shared method body) but for now a TODO to refactor this is probably fine